package.json

```diff
@@ -82,7 +82,7 @@
     "lodash": "^4.14.1",
     "marked": "^0.3.6",
     "mkdirp": "^0.5.1",
-    "moment": "^2.18.1",
+    "moment": "2.24.0",
```

Comment:
We crossed test that didnt break anything?

Reply:
This specifically doesn't matter. We'll check this in the adaptation of SS.